### ISSUE TITLE
Sample has not enough data to estimate library parameters

### ISSUE THREAD

**Issue by wolfylair** [dellytools/delly#198]
Hi,

I am new to Delly and I am running SV calling for my 4 kangaroo species. I have 4 BAM files containing the alignment between kangaroo genome and human genome. The error in the title popped out when i ran the command below:  
 `delly call -x human.hg38.excl.tsv -o delly.bcf -g hs.chr.fa me-1k.above2.bam mf-2k.above2.bam`

Could you please help me with the issue?

Best Regards

Ben

**Comment by tobiasrausch** [comment]
One of your BAM files has less than 10,000 mapped reads. You can use `samtools flagstat input.bam` to get alignment statistics.

**Comment by wolfylair** [comment]
Hi,

Thanks for your quick reply. I checked the number of mapped reads and here is the result:

51305 + 0 in total (QC-passed reads + QC-failed reads)
0 + 0 secondary
51300 + 0 supplementary
0 + 0 duplicates
51305 + 0 mapped (100.00% : N/A)
0 + 0 paired in sequencing
0 + 0 read1
0 + 0 read2
0 + 0 properly paired (N/A : N/A)
0 + 0 with itself and mate mapped
0 + 0 singletons (N/A : N/A)
0 + 0 with mate mapped to a different chr
0 + 0 with mate mapped to a different chr (mapQ>=5)

The assemblies I used for the alignments are chromsome-length Hi-C scaffold with over 600Mb each.  Is there any way to do SV calling using my samples?

Best 

Ben
